scripts/commands/checkLinks.ts

```diff
@@ -11,7 +11,8 @@
 // that they have been altered from the originals.
 
 import { globby } from "globby";
-import { existsSync, readFileSync } from "fs";
+import { existsSync } from "fs";
```

Comment:
Thanks. It'd be great to also remove this by instead pre-reading what all the images are in your `main` function so that you already have all valid files known in your file cache. Then you can remove this call to `existsSync` inside your check function.

Reply:
Thanks! Should be fixed in 17e853d